src/Tests/View/PasswordQuickInput.test.ts

```diff
@@ -22,18 +22,24 @@ import {containsNonAscii} from '../../View/PasswordQuickInput';
 suite('View', function() {
   suite('PasswordQuickInput', function() {
     suite('#containsNonAscii()', function() {
-      test('basics', function() {
+      test('non-ASCII: Korean', function() {
```

Comment:
Please refer to https://github.com/Samsung/ONE-vscode/pull/1154#discussion_r938485139 T.T